src/app/charts/line-graph.component.ts

```diff
@@ -96,20 +96,23 @@ export class LineGraphComponent {
 
     // Set axis scales
     private setAxisScales(): void {
-        var parseTime = D3.timeParse('%Y-%m-%d');
+        // Time scales only recognize annual and daily data
+        var parseTime = D3.timeParse('%Y'); //D3.timeParse('%Y-%m-%d');
```

Comment:
This will only parse yearly aggregations. I suppose we should have some way to know or detect the aggregation level for a given chart. Not sure if we should rely on the client knowing that or not; maybe we could add the date aggregation level to the indicator listing API response and use that.